ocpmodels/datasets/trajectory_lmdb.py

```diff
@@ -129,5 +129,14 @@ def __iter__(self):
 
 
 def data_list_collater(data_list):
+    _padded_indices = []
+    _cumulative_idx = 0
+    for data in data_list:
+        image_pad = (data.edge_index[1, :] == -1).nonzero().view(
+            -1
+        ) + 12 * _cumulative_idx
```

Comment:
@abhshkdz Not sure how we want to handle this for instances where we don't want 12 max neighbors. Don't think we have as much flexibility in the data loader. Unless we make this as part of the `trajectory_lmdb` dataset class and can extract that accordingly.

Reply:
How about something like the following (that doesn't use 12 max neighbors explicitly)?

```
batch = Batch.from_data_list(data_list)
cum_edges = 0
for i, data in enumerate(data_list):
    pad_idx = (data.edge_index[1, :] == -1).nonzero().view(-1)
    n_edges = data.edge_index.shape[-1]
    batch.edge_index[1, cum_edges : cum_edges + n_edges][pad_idx] = -1
    cum_edges += n_edges
return batch
```